.circleci/config.yml

```diff
@@ -95,6 +95,8 @@ jobs:
       - run: {name: 'Refresh Site', command: 'composer site:import -- -c /tmp/db-cache'}
       - save_cache: {name: 'Save DB Cache', key: 'db-{{ checksum "/tmp/db-cache-indicator" }}', paths: ['/tmp/db-cache']}
       - run: {name: 'Wait for Apache', command: 'dockerize -wait tcp://localhost:80 -timeout 5s'}
+      - run: {name: 'Prepare for Nightcrawler', command: 'sleep 60' }
```

Comment:
I'd like to get rid of this sleep before we merge.

Reply:
The sleep 60 was removed from the CircleCI config file.